packages/story-editor/src/app/helpCenter/useHelpCenter/effects.js

```diff
@@ -94,7 +94,10 @@ export function deriveUnreadTipsCount(previous, next) {
 }
 
 export function deriveAutoOpen(previous, next) {
-  if (WEB_STORIES_DISABLE_QUICK_TIPS === 'true') {
+  if (
+    typeof WEB_STORIES_DISABLE_QUICK_TIPS !== 'undefined' &&
+    WEB_STORIES_DISABLE_QUICK_TIPS === 'true'
```

Comment:
Doesn't `WEB_STORIES_DISABLE_QUICK_TIPS === 'true'` already mean that `typeof WEB_STORIES_DISABLE_QUICK_TIPS !== 'undefined'`? 🤔

Reply:
There would  be an error otherwise. Try running `WEB_STORIES_DISABLE_QUICK_TIPS === 'true'` in your dev tools